src/Service.Host/client/src/reducers/index.js

```diff
@@ -132,16 +133,17 @@ const rootReducer = history =>
         automaticPurchaseOrder,
         automaticPurchaseOrderSuggestions,
         batchPurchaseOrderDeliveriesUpload,
+        bomCostReport,
```

Comment:
maybe just a conflict thing but looks like it's putting these back into the wrong order